packages/b/bullet3/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : bullet3
 version    : '3.25'
-release    : 19
+release    : 20
 source     :
     - https://github.com/bulletphysics/bullet3/archive/refs/tags/3.25.tar.gz : c45afb6399e3f68036ddb641c6bf6f552bf332d5ab6be62f7e6c54eda05ceb77
+homepage   : http://bulletphysics.org/
```

Comment:
Better using `https` than `http`. Also this link redirect to `https://pybullet.org/wordpress/`. I think to use the later as homepage, or just `https://pybullet.org`